telegram/constants.py

```diff
@@ -901,3 +964,33 @@ class WebhookLimit(IntEnum):
     """:obj:`int`: Minimum length of the secret token."""
     MAX_SECRET_TOKEN_LENGTH = 256
     """:obj:`int`: Maximum length of the secret token."""
+
+
+class ChatLimit(IntEnum):
+    """This enum contains limitations for :paramref:`telegram.Bot.set_chat_title.title`. The
+    enum members of this enumeration are instances of :class:`int` and can be treated as such.
+
+    .. versionadded:: 20.0
+    """
+
+    __slots__ = ()
+
+    MIN_TITLE_LENGTH = 1
+    """:obj:`int`: Minimum length of a new chat title."""
+    MAX_TITLE_LENGTH = 128
+    """:obj:`int`: Maximum length of a new chat title."""
+
+
+class TopicLimit(IntEnum):
+    """This enum contains limitations for :paramref:`telegram.Bot.edit_forum_topic.name`. The
+    enum members of this enumeration are instances of :class:`int` and can be treated as such.
+
+    .. versionadded:: 20.0
+    """
+
+    __slots__ = ()
+
+    MIN_NAME_LENGTH = 1
+    """:obj:`int`: Minimum length of the topic name."""
+    MAX_NAME_LENGTH = 128
+    """:obj:`int`: Maximum length of the topic name."""
```

Comment:
as per practice adopted in #3351, these docstrings should include references to parameters and methods

Reply:
Done